lib/inv_multi_cg_quda.cpp

```diff
@@ -200,7 +211,7 @@ namespace quda {
       if (param.tol_offset[j] < param.delta) reliable = true;
 
 
-    cudaColorSpinorField *r = new cudaColorSpinorField(b);
+    auto *r = new cudaColorSpinorField(b);
```

Comment:
This reminds me, I need to update the multi-shift solver to use generic `ColorSpinorField` fields (out of scope for this pull).

Reply:
But I like the use of auto 👍